portugues/colabora/semdiferencas/pentefino/sono/cansado/cansado.md

```diff
@@ -0,0 +1 @@
+Finalmente você desiste! Deixa pra lá e resolve procurar um outro app depois, então liga a TV para assistir todas as suas gravações do Big Brother Brasil.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.